en/guidance-updates.rst

```diff
@@ -0,0 +1,104 @@
+IATI Standard Publishing Guidance: How to suggest changes
+=========================================================
+
+
+What information does the IATI Standard Publishing Guidance contain?
+--------------------------------------------------------------------
+
+The `IATI Standard Publishing Guidance <https://iatistandard.org/en/guidance/standard-guidance>`__ helps explain and interpret the IATI Standard. The Standard is written in a technical language and does not contain much of the context of ‘why’ certain data is needed and when and how to use the different IATI elements and attributes. The Standard also contains some gaps and is open to interpretation, such as: should an organisation publish both an original and a revised budget?
+
+The `IATI Standard Publishing Guidance <https://iatistandard.org/en/guidance/standard-guidance>`__ helps address this, by adding context and filling in some of the gaps in the Standard. These pages fit in the **“non-normative”** content category as described below.
+
+- Normative: `Normative content <https://en.wikipedia.org/wiki/Normative#Standards_documents>`__ is the prescriptive part of a standard. It sets the rules to be followed in order to be compliant with the standard, and from which no deviation is permitted.
+
+- Non-normative: Non-normative content is the non-prescriptive, or ‘descriptive’, part of a standard. This may include explanations, illustrations, context, and examples. In the event non-normative content contradicts normative content, the normative content is to be followed.
+
+For a full list of what falls under the normative and non-normative category as part of the IATI standard, see `here <https://en.wikipedia.org/wiki/Normative#Standards_documents>`__.
```

Comment:
@IATI/devs this link will need to be replaced with https://github.com/IATI/IATI-Developer-Documentation/pull/58. Can this be done now or only once the page is live?

Reply:
I will generate the pages for dev.iatistandard.org and see from there if we can determine the link.

Reply:
Fixed this link now. Ready to go!